frontend/apps/marketing/src/components/contentful/collections/peopleCollection/PeopleCollection.tsx

```diff
@@ -142,7 +143,7 @@ const PeopleCollection: React.FC<PeopleCollectionProps> = ({
             ),
           };
         })
-        .sort((a, b) => a.id.localeCompare(b.id)), // Sort alphabetically
+        .sort((a, b) => a?.id?.localeCompare(b?.id)), // Sort alphabetically
```

Comment:
Just to prevent a regression could we add a test to catch the error?

Reply:
Good call, added a test for that and to check the `imageVisibility` prop.